README.md

```diff
@@ -10,36 +10,38 @@
 
 Try .NET provides developers and content authors with tools to create interactive experiences.
 
-There are different flavors of  Try .NET interactive experiences, from the web experience powered by Blazor (as seen on [Microsoft docs](https://docs.microsoft.com/en-us/dotnet/csharp/tutorials/intro-to-csharp/hello-world)), to interactive documentation for .NET Core with the [dotnet try global tool](DotNetTry.md), to [.NET Jupyter Notebooks](Notebook.md). 
+There are different experiencs of  Try .NET interactive experiences, from the web experience powered by Blazor (as seen on [Microsoft docs](https://docs.microsoft.com/en-us/dotnet/csharp/tutorials/intro-to-csharp/hello-world)), to interactive documentation for .NET Core with the [dotnet try global tool](DotNetTry.md). 
 
-## Flavors of Try .NET 
+## Experiences  
 
-### Try .NET Online
+### Online, powered by Blazor
 
-<img src="https://user-images.githubusercontent.com/2546640/57144765-c850cc00-6d8f-11e9-982d-50d2b6dc3591.gif" width="60%">
+Microsoft Docs uses Try .NET to create interactive documentation. Users can run and edit code all in the browser.
 
-### Interactive .NET Core documentation with the `dotnet try` global tool
+_**Please Note**: At the moment, the Try .NET online (`trydotnet.js API`) is currently only used internally at Microsoft as seen on docs and [.NET page](https://dotnet.microsoft.com/learn/dotnet/in-browser-tutorial/1)._
```

Comment:
Maybe also worth mentioning the Microsoft docs documentation about how to create interactive docs? https://docs.microsoft.com/en-us/contribute/code-in-docs#interactive-code-snippets

Reply:
docs is still make some improvements on this experience. I would like this to be resolved before linking it here.